src/cares_wrap.cc

```diff
@@ -1806,10 +1825,14 @@ static void Query(const FunctionCallbackInfo<Value>& args) {
   CHECK_EQ(false, args.IsConstructCall());
   CHECK(args[0]->IsObject());
   CHECK(args[1]->IsString());
+  CHECK(args[2]->IsBoolean());
 
   Local<Object> req_wrap_obj = args[0].As<Object>();
   Local<String> string = args[1].As<String>();
-  Wrap* wrap = new Wrap(channel, req_wrap_obj);
+  bool is_search = args[2]->IsTrue();
```

Comment:
Are these two interchangeable statements?
- `bool is_search = args[2]->IsTrue();`
- `bool is_search = args[2]->Value();`

Or is one of them preferred / idiomatic over the other?

Reply:
`IsTrue()` should be fine.